chainerrl/functions/__init__.py

```diff
@@ -1,3 +1,5 @@
+from chainerrl.functions.mul_add import muladd  # NOQA
+from chainerrl.functions.mul_add import MulAdd  # NOQA
```

Comment:
Chainer stopped exporting `FunctionNode`.

Reply:
Got it. Removed the line.